src/api/PublishDialog.js

```diff
@@ -107,7 +107,18 @@ export default class PublishDialog extends Component {
         {contentAttributions && (
           <FormField>
             <label>Model Attribution:</label>
-            <p>{contentAttributions.map(a => `${a.name} by ${a.author}\n`)}</p>
+            <ul>
+              {contentAttributions.map(
+                (a, i) =>
+                  a.author &&
```

Comment:
We check for author down below as well. I think maybe we just want to check for title here?

Reply:
As long as we want to so attribution id just one is present. Don't know if it makes sense to show only the title without knowing the author or just the author without knowing to which asset is referring to.